### DISCUSSION TITLE
公式检测模型的准确度有待提升

### DISCUSSION THREAD

**Discussion by lingyezhixing** [OleehyO/TexTeller#52] [feedback]
分段识别中有些变量字母会消失，猜测是公式识别时漏掉了这些单一字母。

**Comment by lingyezhixing** [comment]
![屏幕截图 2024-05-30 203359](https://github.com/OleehyO/TexTeller/assets/144504450/e99e36f3-f2c7-4697-8d0e-4ea474aac113)
使用段落识别这张图片时行列式识别出错，猜测是公式检测模型没有标出行列式，直接使用了文本ocr进行的识别。

**Comment by OleehyO** [comment]
是的，这个case是公式检测没有识别出整个行列式。

另外建议这种单行的图可以直接丢到公式识别模型里识别并使用稍微大一点的number of beams(比如3)


<img width="805" alt="截屏2024-06-07 13 35 24" src="https://github.com/OleehyO/TexTeller/assets/56267907/4eda2eea-9763-422f-bef3-5d9ac5b8ceba">
